### PR TITLE
docs: document global typography class prompt

### PR DESCRIPTION
**Pull request by djbrown**
add documentation for #17602

### CODE REVIEW

File: guides/getting-started.md [hunk 1]
```diff
@@ -22,7 +22,11 @@ The `ng add` command will install Angular Material, the [Component Dev Kit (CDK)
 
    You can choose from [prebuilt material design themes](https://material.angular.io/guide/theming#using-a-pre-built-theme) or set up an extensible [custom theme](https://material.angular.io/guide/theming#defining-a-custom-theme).
 
-2. Set up browser animations for Angular Material:
+2. Set up global Angular Material typography styles:
+
+   Wether to apply the global [Typography](https://material.angular.io/guide/typography) style to your application.
```
Comment (crisbeto) [inline guides/getting-started.md:27]: Wether - Whether
Typography -> typography
style -> styles

**Review by crisbeto — COMMENTED** [review]
Also it looks like the lint check for the commit message is failing. You can try changing it to:

```
docs: document global typography class prompt
```

**Comment by djbrown** [comment]
I fixed the typos as well as the commit and PR message

**Review by crisbeto — APPROVED** [review]
LGTM

**Pull request merged**